### ISSUE TITLE
500 error after update to  Core 2.2.7.2

### ISSUE THREAD

**Issue by CentricStraaten** [PiranhaCMS/piranha.core#47]
If have update the NuGet Packackes tot 2.2.7.2. All my packages are up-to-date, but my project is not working anymore.

I receive  a 500 Error: 

> __DynamicModule_Piranha.Web.ApplicationModule, Piranha, Version=2.2.6243.15973, Culture=neutral, PublicKeyToken=null_4c0a1dc4-8c7c-4864-9797-b0b2f34f30f9

Does anyone know what is going wrong here?

I have following packages installed
`<?xml version="1.0" encoding="utf-8"?>
<packages>
  <package id="Antlr" version="3.5.0.2" targetFramework="net452" />
  <package id="AutoMapper" version="5.2.0" targetFramework="net452" />
  <package id="bootstrap.less" version="3.3.7" targetFramework="net452" />
  <package id="EntityFramework" version="6.1.3" targetFramework="net451" />
  <package id="FontAwesome" version="4.7.0" targetFramework="net452" />
  <package id="jQuery" version="3.1.1" targetFramework="net452" />
  <package id="jQuery.Validation" version="1.16.0" targetFramework="net452" />
  <package id="MarkdownSharp" version="1.13.0.0" targetFramework="net451" />
  <package id="Microsoft.ApplicationInsights" version="2.2.0" targetFramework="net452" />
  <package id="Microsoft.ApplicationInsights.Agent.Intercept" version="2.0.7" targetFramework="net452" />
  <package id="Microsoft.ApplicationInsights.DependencyCollector" version="2.2.0" targetFramework="net452" />
  <package id="Microsoft.ApplicationInsights.JavaScript" version="0.15.0-build58334" targetFramework="net451" />
  <package id="Microsoft.ApplicationInsights.PerfCounterCollector" version="2.2.0" targetFramework="net452" />
  <package id="Microsoft.ApplicationInsights.Web" version="2.2.0" targetFramework="net452" />
  <package id="Microsoft.ApplicationInsights.WindowsServer" version="2.2.0" targetFramework="net452" />
  <package id="Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel" version="2.2.0" targetFramework="net452" />
  <package id="Microsoft.AspNet.Mvc" version="5.2.3" targetFramework="net451" />
  <package id="Microsoft.AspNet.Razor" version="3.2.3" targetFramework="net451" />
  <package id="Microsoft.AspNet.Web.Optimization" version="1.1.3" targetFramework="net451" />
  <package id="Microsoft.AspNet.WebApi" version="5.2.3" targetFramework="net452" />
  <package id="Microsoft.AspNet.WebApi.Client" version="5.2.3" targetFramework="net452" />
  <package id="Microsoft.AspNet.WebApi.Core" version="5.2.3" targetFramework="net452" />
  <package id="Microsoft.AspNet.WebApi.WebHost" version="5.2.3" targetFramework="net452" />
  <package id="Microsoft.AspNet.WebPages" version="3.2.3" targetFramework="net451" />
  <package id="Microsoft.jQuery.Unobtrusive.Validation" version="3.2.3" targetFramework="net451" />
  <package id="Microsoft.Web.Infrastructure" version="1.0.0.0" targetFramework="net451" />
  <package id="Modernizr" version="2.8.3" targetFramework="net452" />
  <package id="Newtonsoft.Json" version="9.0.1" targetFramework="net452" />
  <package id="PagedList" version="1.17.0.0" targetFramework="net452" />
  <package id="PagedList.Mvc" version="4.5.0.0" targetFramework="net452" />
  <package id="PiranhaCMS.Core" version="2.2.7.2" targetFramework="net452" />
  <package id="PiranhaCMS.Mvc" version="2.2.6" targetFramework="net452" />
  <package id="Respond" version="1.4.2" targetFramework="net452" />
  <package id="Twitter.Bootstrap.Less" version="3.3.6" targetFramework="net452" />
  <package id="WebActivatorEx" version="2.2.0" targetFramework="net452" />
  <package id="WebGrease" version="1.6.0" targetFramework="net452" />
</packages>`

**Comment by CentricStraaten** [comment]
I found the cause. EF Migration is trying to update the website, but fails. Whenever is assign a new database everything works. How do i solve this?

**Comment by tidyui** [comment]
Sorry for the late response, but I think your issue should be in the Piranha repo instead. In regards to your error I'll try to reproduce it. Piranha doesn't use EF migrations, it uses embedded SQL-scripts for scheme updates, so just the fact that EF is trying to update it points to something being wrong.

**Comment by CentricStraaten** [comment]
Okay, you closed the ticket but the issue exists. Should i post it in the Piranha repo then?

**Comment by CentricStraaten** [comment]
`ERROR [2017-04-06 19:56:45] Origin [WebPiranha.BeginRequest] Message [Unhandled exception]
System.Data.SqlClient.SqlException (0x80131904): Invalid column name 'pagetemplate_is_block'.
   at System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   at System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   at System.Data.SqlClient.SqlDataReader.TryConsumeMetaData()
   at System.Data.SqlClient.SqlDataReader.get_MetaData()
   at System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString, Boolean isInternal, Boolean forDescribeParameterEncryption)
   at System.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean async, Int32 timeout, Task& task, Boolean asyncWrite, Boolean inRetry, SqlDataReader ds, Boolean describeParameterEncryptionRequest)
   at System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method, TaskCompletionSource`1 completion, Int32 timeout, Task& task, Boolean& usedCache, Boolean asyncWrite, Boolean inRetry)
   at System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method)
   at System.Data.SqlClient.SqlCommand.ExecuteReader(CommandBehavior behavior, String method)
   at System.Data.SqlClient.SqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   at System.Data.Common.DbCommand.System.Data.IDbCommand.ExecuteReader(CommandBehavior behavior)
   at Piranha.Data.ActiveRecord`1.Query(String query, Object[] args)
   at Piranha.Data.ActiveRecord`1.GetFields(String fields, String where, Object[] args)
   at Piranha.Data.ActiveRecord`1.GetSingle(String where, Object[] args)
   at Piranha.Models.Page.GetStartpage(Boolean draft)
   at Piranha.Mvc.RouteHandler.HandleStartpage(HttpContext context)
   at Piranha.Web.Handlers.PermalinkHandler.HandleRequest(HttpContext context, Boolean draft, String[] args)
   at Piranha.Web.Handlers.PermalinkHandler.HandleRequest(HttpContext context, String[] args)
   at Piranha.WebPages.WebPiranha.BeginRequest(HttpContext context)
ClientConnectionId:d1078343-11c2-4671-9abe-118b948db297
Error Number:207,State:1,Class:16`

**Comment by tidyui** [comment]
This error actually looks like has nothing to do with EF as it is a SqlError from `ActiveRecord` which doesn't use EF. So basically it's trying to access the column `pagetemplate_is_block` which is fairly new (v2.2.5) but for some reason the database is not update.

From which version are you updating?

And the reason I close the issue was that I commented on it 28 days ago and never heard anything. From experience this usually means that the problem has been solved :)

/Håkan
